csharp/src/Ice/Reference.cs

```diff
@@ -672,6 +709,7 @@ static void StartQueryOption(StringBuilder sb, ref bool firstOption)
             string facet,
             Identity identity,
             InvocationMode invocationMode,
+            TimeSpan invocationTimeout,
```

Comment:
Why do we have an explicit parameter here? AFAICT the callers always pass the communicator's default invocation timeout so shouldn't we get rid of this parameter and pass the communicator's default value like other parameters such as the locator cache timeout?

Reply:
Removed it